### PR TITLE
feat: clean up project

### PR DESCRIPTION
**Pull request by damianopetrungaro**

### CODE REVIEW

**Comment by damianopetrungaro** [comment]
@moon-php/committee Please review it 😄

File: .scrutinizer.yml [hunk 1]
```diff
@@ -1,14 +0,0 @@
-build:
```
Comment (mereba) [inline .scrutinizer.yml:1]: @damianopetrungaro Why are you removing this file? I can see it is handling code coverage reports?

Comment (damianopetrungaro) [reply .scrutinizer.yml:1]: I migrated from `scrutinizier` to `travis` (it is faster and way easier to interact with)
We can add the tests coverage using `travis` too.

I am free after 6/7 PM  to do it 😄

**Comment by damianopetrungaro** [comment]
@mereba @ElenaForester  here we go 😄 
Review it and 🤞 Travis-CI is configured properly 😄

**Review by ElenaForester — APPROVED** [review]
Seems ok

**Pull request merged**